galley/pkg/config/analysis/analyzers/testdata/authorizationpolicies.yaml

```diff
@@ -0,0 +1,935 @@
+              f:runAsGroup: {}
+              f:runAsNonRoot: {}
+              f:runAsUser: {}
+            f:terminationMessagePath: {}
+            f:terminationMessagePolicy: {}
+        f:restartPolicy: {}
+        f:schedulerName: {}
+        f:securityContext:
+          .: {}
+          f:fsGroup: {}
+        f:serviceAccount: {}
+        f:serviceAccountName: {}
+        f:terminationGracePeriodSeconds: {}
+        f:volumes:
+          .: {}
+          k:{"name":"istio-data"}:
+            .: {}
+            f:emptyDir: {}
+            f:name: {}
+          k:{"name":"istio-envoy"}:
+            .: {}
+            f:emptyDir:
+              .: {}
+              f:medium: {}
+            f:name: {}
+          k:{"name":"istio-podinfo"}:
+            .: {}
+            f:downwardAPI:
+              .: {}
+              f:defaultMode: {}
+              f:items: {}
+            f:name: {}
+          k:{"name":"istiod-ca-cert"}:
+            .: {}
+            f:configMap:
+              .: {}
+              f:defaultMode: {}
+              f:name: {}
+            f:name: {}
+    manager: kube-controller-manager
+    operation: Update
+    time: "2020-06-17T09:14:04Z"
+  - apiVersion: v1
+    fieldsType: FieldsV1
+    fieldsV1:
+      f:status:
+        f:conditions:
+          k:{"type":"ContainersReady"}:
+            .: {}
+            f:lastProbeTime: {}
+            f:lastTransitionTime: {}
+            f:status: {}
+            f:type: {}
+          k:{"type":"Initialized"}:
+            .: {}
+            f:lastProbeTime: {}
+            f:lastTransitionTime: {}
+            f:status: {}
+            f:type: {}
+          k:{"type":"Ready"}:
+            .: {}
+            f:lastProbeTime: {}
+            f:lastTransitionTime: {}
+            f:status: {}
+            f:type: {}
+        f:containerStatuses: {}
+        f:hostIP: {}
+        f:initContainerStatuses: {}
+        f:phase: {}
+        f:podIP: {}
+        f:podIPs:
+          .: {}
+          k:{"ip":"172.17.0.14"}:
+            .: {}
+            f:ip: {}
+        f:startTime: {}
+    manager: kubelet
+    operation: Update
+    time: "2020-06-17T09:14:39Z"
+  name: httpbin-55bf89f8c9-wzfrh
+  namespace: httpbin
+  ownerReferences:
+  - apiVersion: apps/v1
+    blockOwnerDeletion: true
+    controller: true
+    kind: ReplicaSet
+    name: httpbin-55bf89f8c9
+    uid: 89cf9c4c-a1d5-4521-923d-3c50791865aa
+  resourceVersion: "124086"
+  selfLink: /api/v1/namespaces/httpbin/pods/httpbin-55bf89f8c9-wzfrh
+  uid: cf33bf5f-4dec-4d71-a730-65d5ea2dd0b6
+spec:
+  containers:
+  - image: docker.io/kennethreitz/httpbin
+    imagePullPolicy: IfNotPresent
+    name: httpbin
+    ports:
+    - containerPort: 80
+      protocol: TCP
+    resources: {}
+    terminationMessagePath: /dev/termination-log
+    terminationMessagePolicy: File
+    volumeMounts:
+    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+      name: httpbin-token-dlcm2
+      readOnly: true
+  - args:
```

Comment:
This is injecting the proxy manually, why not using auto-inject?

Reply:
Change it. Also took logic from previous analyzers to check not only containers but namespace and pod annotations.